lib/ansible/modules/shell.py

```diff
@@ -179,7 +179,7 @@
     description: The command standard output
     returned: always
     type: str
-    sample: 'Clustering node rabbit@slave1 with rabbit@master …'
+    sample: ' 20:45:08 up 19 days,  9:39,  2 users,  load average: 0.90, 1.04, 1.39'
 stderr:
     description: The command standard error
     returned: always
```

Comment:
GitHub won't let me add the inline comment in the right place, but a few lines down it's probably worth replacing line 192 with `sample: uptime`